hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -211,7 +211,7 @@
                         network_config:
                         - type: interface
                           name: nic1
-                          use_dhcp: true
+                          use_dhcp: false
```

Comment:
nic1 is the primary nic which zuul connects to and disabling dhcp there would cause issues. so the failing job looks related to this.